nifi-nar-bundles/nifi-ignite-bundle/nifi-ignite-nar/src/main/resources/META-INF/NOTICE

```diff
@@ -0,0 +1,26 @@
+nifi-ignite-nar
```

Comment:
Any NOTICE information in this Notice file that is not in the nifi-assembly NOTICE,  should be added to the assembly NOTICE as well